Source/ACE.Server/WorldObjects/Vendor.cs

```diff
@@ -293,7 +293,7 @@ public void SellItems_ValidateTransaction(Player player, List<WorldObject> items
                     wo.CurrentWieldedLocation = null;
                     wo.Placement = ACE.Entity.Enum.Placement.Resting;
 
-                    if ((wo.GetProperty(PropertyBool.DestroyOnSell) ?? false) == true)
+                    if (!(wo.GetProperty(PropertyBool.DestroyOnSell) ?? false))
```

Comment:
should we also just Destroy the item somewhere in the code?

Reply:
This is a good question, even for items that aren't DestroyOnSell

When a player sells a regular item to a vendor, it gets added to their uniqueItems, which only persists until the next server restart. When the server is restarted, these items are essentially in limbo, with no references to them anywhere, but still saved in the database...